pkgs/applications/networking/cluster/k3s/update-script.sh

```diff
@@ -76,15 +76,16 @@ EOF
 mv chart-versions.nix.update chart-versions.nix
 
 # Concatenate all sha256sums, one entry per line
-SHA256_HASHES="$(curl -L "https://github.com/k3s-io/k3s/releases/download/v${K3S_VERSION}/sha256sum-amd64.txt")
-    \n$(curl -L "https://github.com/k3s-io/k3s/releases/download/v${K3S_VERSION}/sha256sum-arm64.txt")
-    \n$(curl -L "https://github.com/k3s-io/k3s/releases/download/v${K3S_VERSION}/sha256sum-arm.txt")"
+SHA256_HASHES="\
```

Comment:
```suggestion
SHA256_HASHES="\
```
The text file was not being properly generated, it was including "extra spaces" and "\n".